src/NativeRopeContext.cpp

```diff
@@ -144,6 +143,21 @@ void NativeRopeContext::_simulate_velocities(double delta)
     std::swap(oldpoints, points);
 }
 
+void NativeRopeContext::_simulate_wind(PackedVector2Array* velocities) const
+{
+    if (wind.is_null() || !wind->get_enable_wind() || wind->get_noise().is_null())
+        return;
+
+    const float time = (float)Time::get_singleton()->get_ticks_msec();
+    const Vector2 wind_velocity = wind->get_direction_vector() * wind->get_wind_strength();
+    for (int i = 0; i < points.size(); ++i)
+    {
+        const float noise = wind->get_noise()->get_noise_3d(points[i].x, points[i].y, time);
+        const Vector2 noise_velocity = wind->get_direction_vector().orthogonal() * (noise * wind->get_oscillation_strength());
```

Comment:
- `wind->get_direction_vector().orthogonal() * wind->get_oscillation_strength()` can be cached just like `wind_velocity`.
- Linebreak between the variables and the for loop